internal/bminventory/inventory.go

```diff
@@ -6202,14 +6202,15 @@ func (b *bareMetalInventory) V2RegisterHost(ctx context.Context, params installe
 	defaultRole := models.HostRoleAutoAssign
 
 	host := &models.Host{
-		ID:                    params.NewHostParams.HostID,
-		Href:                  swag.String(url.String()),
-		Kind:                  kind,
-		CheckedInAt:           strfmt.DateTime(time.Now()),
-		DiscoveryAgentVersion: params.NewHostParams.DiscoveryAgentVersion,
-		UserName:              ocm.UserNameFromContext(ctx),
-		Role:                  defaultRole,
-		InfraEnvID:            *infraEnv.ID,
+		ID:                       params.NewHostParams.HostID,
+		Href:                     swag.String(url.String()),
+		Kind:                     kind,
+		CheckedInAt:              strfmt.DateTime(time.Now()),
+		DiscoveryAgentVersion:    params.NewHostParams.DiscoveryAgentVersion,
+		UserName:                 ocm.UserNameFromContext(ctx),
+		Role:                     defaultRole,
+		InfraEnvID:               *infraEnv.ID,
+		IgnitionEndpointTokenSet: false,
```

Comment:
Why do we care if the token is set?
Unlike pull secret that is mandatory for installation ignition endpoint token is optional